tools/Vigie-Chiro/.shed.yml

```diff
@@ -0,0 +1,68 @@
+categories:
+- ZZ-test1 
+description: Tools created by the vigiechiro team to analyses and identify chiro sounds files.
+
+long_description: |
+  TODO
+
+homepage_url: https://www.vigienature-ecole.fr/les-observatoires/le-protocole-vigie-chiro 
+name: vigiechiro
+owner: mnhn65mo
+remote_repository_url: https://github.com/Alanamosse/Galaxy-E/tree/master/tools/Vigie-Chiro
+type: unrestricted
+
+#auto_tool_repositories:
```

Comment:
why not this one?

Reply:
I tried auto_tool_repo, it creates repo for tool1 tool2 etc but they all get every Rscripts and csv files attached (some not necessary). I think it's not optimal, so I just manually listed files that need to be in every repo..